automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -28,31 +33,41 @@ trait GPAllocFixtureSpec extends FixtureAnyFreeSpecLike with Retries {
       outcome
     }
 
-    sys.props.get(gpallocProjectKey) match {
-      case None                                            => throw new RuntimeException("leonardo.billingProject system property is not set")
+    sys.props.get(workspaceNamespaceKey) match {
```

Comment:
This `match` statement should account for all cases. 

Actually maybe we should just remove this and put the `gpallocErrorPrefix` in the `googleProjectKey` property, and check for it down below.

Reply:
I think googleProjectKey here is just a string variable. Does it still make sense to turn it into a property?

Reply:
Yeah `googleProjectKey` is a string key for a Java system property. We're overloading the property a bit -- the property value with either contain a google project name, _or_ an error message if we encountered an error generating the project. 

My comment is just to remove the `sys.props.get(workspaceNamespaceKey) match …` and just do:
```
sys.props.get(googleProjectKey) match {
      case None => throw new RuntimeException("leonardo.googleProject system property is not set")
      case Some(msg) if msg.startsWith(gpallocErrorPrefix) => throw new RuntimeException(msg)
      case Some(googleProjectId) => // run the test
    }
```

Then in the `beforeAll` we put the error message in the `googleProjectKey` property value instead.

Reply:
Thanks for explaining, made the changes